test/javascript/erlang/erlang_test.mjs

```
@@ -3960,6 +5417,77 @@ describe("Erlang", () => {
+
+    it("keeps zero integer unchanged", () => {
+      const result = testedFun(integer0);
+
+      assert.deepStrictEqual(result, integer0);
+    });
+
+    it("demonstrates floating-point precision limits for large numbers", () => {
+      // eslint-disable-next-line no-loss-of-precision
+      const result = testedFun(Type.float(36028797018963969.0));
+
+      const expected = Type.integer(36028797018963968n);
```

Biome (2.1.2)

[error] This number literal will lose precision at runtime.

The value at runtime will be 36028797018963970

(lint/correctness/noPrecisionLoss)